spac-flow-docker-snapshot/README.md

```diff
@@ -0,0 +1,48 @@
+# spac-flow-docker-snapshot
+
+Allows to build docker image with the follow tags:
+
+- snapshot
+- snapshot-(branch name)
+
+and if runned manually, create a fake tag called `develop-snapshot`
+
+## how to use
+
+```yaml
+name: SPAC Snapshot docker build and push
```

Comment:
@diegolagospagopa can we use a less project-bounded name?

Reply:
Right now this flow is tailor made for SPAC, if it work as excepted can be generalize more